smart-table-module/js/Table.js

```diff
@@ -176,11 +185,15 @@
 
             /**
              * insert a new column in scope.collection at index or push at the end if no index
+             * call setDefaultSortColumn to set lastCoulmnSort to already sorted column
              * @param columnConfig column configuration used to instantiate the new Column
              * @param index where to insert the column (at the end if not specified)
              */
             this.insertColumn = function (columnConfig, index) {
                 var column = new Column(columnConfig);
+                if (angular.isDefined(column.reverse)) {
```

Comment:
what are the possible values for `column.reverse`? is `angular.isDefined` robust enough to do what we want?